tests/e2e/specs/product-editor/block-integration.test.js

```diff
@@ -13,6 +13,7 @@ test.use( { storageState: process.env.ADMINSTATE } );
 test.describe.configure( { mode: 'serial' } );
 
 test.describe( 'Product Block Editor integration', () => {
+	test.skip( 'Experimental block editor is deprecated in v10.1.0' );
```

Comment:
Should we conditionally skip these based on the version of the plugin being tested, or should we remove these altogether for now?

Reply:
Considering that WooCommerce 10.1 has only removed its feature option from the UI, it can remain as is, with tests skipped but not removed. Future actions could be aligned with WooCommerce Core's deprecation or adjustments to its direction and progress.